### ISSUE TITLE
BL

### ISSUE THREAD

**Issue by peekpt** [albert21martins/SolderingStation-master#1]
Hello, I am sorry to bother you, I am trying to design my own board from your schematic.

I see the net 'BL'  in the pin 5 and ,the display header and the source of mosfet.

I also noticed that pin 7 is connected to control  mosfet  (T3) of the Backlight. 
I am confused trying to figured out the propose, but I think you meant to connect the gate to pin 5 instead of 7. I don't see nothing referenced to pin 7 on the program code.

Please help thank you

**Comment by albert21martins** [comment]
The backligtht is controlled only with the pin 5. Pin 7 was only for future purposes. For the moment is not implemented in the code. When I have time I will remake it all.

**Comment by peekpt** [comment]
Anyway the mosfet will not work properly, because you're loading in the source, you will have 5v on the source and you can't put Vgs over 5v. ( you need a P mosfet there)   I started redrawing from your schematic, I am changing some things, like rotary encoder switch , added buzzer for beep notification and a  GX-16 iron plug which will have a pin for a reed switch ( you can have a stand with a magnet, wherever you want, no need for extra wire.  Thank you! :)

http://imgur.com/a/vhPdz

**Comment by albert21martins** [comment]
Yeah thanks for the tip. I find you schematic really cool. How do you plan to place the reed sensor. Btw I will steal the connector you use :)

**Comment by peekpt** [comment]
The reed will be inside the tip connector like original. Maybe I do a simple tube, or buy a cheap plastic soldering iron from china take the guts out and use only the tube part. I don't know yet.
http://imgur.com/a/PVUxv
I am still working on the pcb but...

...I am a bit 'jaw dropped'  with this hackaday project  https://hackaday.io/project/18899-rt-soldering-pen
